src/block.tsx

```diff
@@ -19,10 +19,22 @@ import { classNames, getTextContent, getListNumber } from "./utils";
 
 export const createRenderChildText = (
   customDecoratorComponents?: CustomDecoratorComponents
-) => (properties: DecorationType[]) => {
+) => (properties: DecorationType[], blocks: BlockMapType) => {
   return properties?.map(([text, decorations], i) => {
     if (!decorations) {
-      return <React.Fragment key={i}>{text}</React.Fragment>;
+      return (
+        <React.Fragment key={i}>
+          {text.split(/(\n)/).map((t, index) => (
+            <React.Fragment key={index}>
+              {t !== "\n" ? t : <br />}
+            </React.Fragment>
+          ))}
+        </React.Fragment>
+      );
```

Comment:
This looks a bit hacky, should we wrap it inside `span` and use `innerHTML`?

Reply:
_[👉 comment](https://github.com/splitbee/react-notion/pull/40#issuecomment-820567919)_